help/changelog.yaml

```diff
@@ -35,3 +35,4 @@
   - Added <i>Artifact2Image</i> procedure.
   - Discover procedure is <i>Deprecated</i>. Use the <i>Import Microservices</i> procedure to create microservice models based on the given Docker Compose file contents
   - Added support for retrieving and creating the plugin configurations through the <i>Configurations</i> option on the application process step and the pipeline stage task editors.
+1.4.0:
```

Comment:
Add note that 'Fixed the report link for unsupported tags that were not processed by the 'Import Microservices' procedure.'